Examples/sampleseleniumtest.py

```diff
@@ -1,42 +1,195 @@
+        self.assertTrue(self.page_source.startswith("<!DOCTYPE html"))
+        self.locator_check(
+            PageObjects.links_all.get("page_footer"))
+
+        self.assertTrue(self.by_value, 'css selector')
+        self.scroll_to_footer()
+        self.hover_on_element(
+            PageObjects.links_all.get("page_footer"))
+        self.wait_for_element(
+            PageObjects.links_all.get("page_footer"))
+        self.hover_on_click(
+            PageObjects.links_all.get("page_footer"))
+        window_before = self.driver.window_handles[0]
+        window_after = self.driver.window_handles[1]
+        self.switch_to_window(window_after)
+        self.reload_page()
+        url = self.get_location()
+        self.assertTrue(url.startswith("http://elementalselenium.com/"))
+        self.text = self.get_text(PageObjects.labels_all.get('header'))
+        self.assertTrue(str(self.text), 'Elemental Selenium')
+        self.driver.close()
+        self.switch_to_window(window_before)
+        self.wait_for_element(
+            PageObjects.links_all.get("checkboxes"))
+        self.click(
+            PageObjects.links_all.get("checkboxes"))
+        self.wait_for_element(
+            PageObjects.checkBox_all.get("checkbox1"))
+        self.click(PageObjects.
+                   checkBox_all.get("checkbox1"))
+        web_elements = self.find_elements(
+            PageObjects.checkBox_all.get("checkbox1"))
+        self.assertTrue(web_elements[0].is_selected())
+        self.go_back()
+        self.page_readiness_wait()
+        self.scroll_to_element(PageObjects.links_all.get('alerts'))
+        self.click(
+            PageObjects.links_all.get('alerts'))
+        self.reload_page()
+        self.click(PageObjects.buttons_all.get('js_alert'))
+        self.switch_to_alert()
+        time.sleep(5)
+        self.wait_and_accept_alert()
+        self.go_back()
+        self.go_forward()
+        self.click(PageObjects.buttons_all.get('js_confirm_alert'))
+        self.switch_to_alert()
+        time.sleep(5)
+        self.wait_and_reject_alert()
+        self.click(PageObjects.buttons_all.get('js_prompt_alert'))
+        self.switch_to_alert()
+        time.sleep(5)
+        self.wait_and_reject_alert()
+        self.go_back()
+        self.click(PageObjects.links_all.get('frames'))
+        self.click(PageObjects.links_all.get('iframe'))
+        self.page_readiness_wait()
+        self.switch_to_frame('mce_0_ifr')
+        self.clear_text(PageObjects.text_boxes.get('text_in_frame'))
+        self.switch_to_active_element()
+        self.go_back()
+        self.go_back()
+        self.page_readiness_wait()
+        time.sleep(5)
+        self.click(PageObjects.links_all.get('dynamic_controls'))
+        self.click(PageObjects.buttons_all.get('enable_btn'))
+        time.sleep(5)
+        self.send_keys(PageObjects.text_boxes.get('text_box'))
+        time.sleep(5)
+        self.switch_to_default_content()
+        path = os.getcwd()
```

Comment:
can be merged and re written as self.capture_screenshot(os.getcwd()+"\\example_Screenshot1.png")